lib/MusicBrainz/Server/Entity/Tag.pm

```diff
@@ -10,16 +10,30 @@ has 'name' => (
     isa => 'Str'
 );
 
-sub TO_JSON {
-    return shift->name;
-}
+has 'genre_id' => (
+    is => 'rw',
+    isa => 'Int'
+);
+
+has 'genre' => (
+    is => 'rw',
+    isa => 'Genre'
```

Comment:
Shouldn’t that be optional/maybe?

Reply:
I thought about that, but Entity::Artist for example has 

`has 'begin_area_id' => (
    is => 'rw',
    isa => 'Int'
);

has 'begin_area' => (
    is => 'rw',
    isa => 'Area'
);`

Reply:
You could say `Maybe[Genre]` or `Maybe[Area]` are more correct since not all tags have a genre and not all artists have a begin area, but it wouldn't cause any issue without `Maybe` unless you pass `genre => undef` or `begin_area => undef` to the constructor, which you wouldn't normally do. (Moose differentiates between an attribute being unset, and being set to undef. Since these are `rw`, you don't need to set them.)